### DISCUSSION TITLE
Where do I put my Embeddings? in Textual Inversion folder? but where is it located?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#1962] [Q&A] [answered]
just what the question says

modules Textual Inversion doesn't work

**Comment by brunogcar** [comment]
../automatic/models/embeddings

**Reply by weirdlighthouse** [reply]
![tdgfdgdggdgg](https://github.com/vladmandic/automatic/assets/140368252/c021e1a8-786b-4432-ad88-cb3aa4cb8ecc)

where do see embeddings folder bro?

**Reply by weirdlighthouse** [reply, marked as answer]
okay just created new folder and it started working idk why it wasn't there from the start
